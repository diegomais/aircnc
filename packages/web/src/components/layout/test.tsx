import React from 'react'
import { render } from '@testing-library/react'
import { Layout } from '.'

describe('<LayoutComponent />', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Layout>{}</Layout>)
    expect(asFragment()).toMatchSnapshot()
  })

  it('should show a logo image', () => {
    const { getByAltText } = render(<Layout>{}</Layout>)
    expect(getByAltText(/aircnc/i)).toBeInTheDocument()
  })

  it('should render children element', () => {
    const { getByText } = render(
      <Layout>
        <p>some text</p>
      </Layout>
    )
    expect(getByText(/some text/i)).toBeInTheDocument()
  })
})
