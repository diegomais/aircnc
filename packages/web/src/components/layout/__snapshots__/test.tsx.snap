// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutComponent /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <img
      alt="aircnc"
      src="images/logo.svg"
    />
    <main
      class="main"
    />
  </div>
</DocumentFragment>
`;
